airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/ConfigRepositoryE2EReadWriteTest.java

```diff
@@ -609,4 +610,37 @@ private void insertCatalogFetchEvent(final DSLContext ctx, final UUID sourceId,
         .execute();
   }
 
+  private static void writeActorCatalog(final Database database, final List<ActorCatalog> configs) throws IOException, SQLException {
```

Comment:
moved helpers from `DatabaseConfigPersistence` that were only used in these tests.